src/app/shared/dialog-box/dialog-box.component.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
once we remove reference to angular Material. Remove the references from the tests too